pkg/asset/agent/configimage/configiso.go

```diff
@@ -0,0 +1,159 @@
+package configimage
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/vincent-petithory/dataurl"
+
+	"github.com/openshift/assisted-image-service/pkg/isoeditor"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/agent/image"
+)
+
+const (
+	configImageFilename = "agentconfig.noarch.iso"
+	configCpioArchive   = "config.gz"
+
+	pullSecretManifest          string = "/etc/assisted/manifests/pull-secret.yaml" //nolint:gosec // not hardcoded credentials
+	nmStateConfigManifest       string = "/etc/assisted/manifests/nmstateconfig.yaml"
+	clusterDeploymentManifest   string = "/etc/assisted/manifests/cluster-deployment.yaml"
+	clusterImageSetManifest     string = "/etc/assisted/manifests/cluster-image-set.yaml"
+	agentClusterInstallManifest string = "/etc/assisted/manifests/agent-cluster-install.yaml"
+	extraManifests              string = "/etc/assisted/extra-manifests"
+	hostConfigFiles             string = "/etc/assisted/hostconfig"
+	hostNameFiles               string = "/etc/assisted/hostnames"
+	kubeadminPasswordHash       string = "/opt/agent/tls/kubeadmin-password.hash" //nolint:gosec // not hardcoded credentials
```

Comment:
IIUC, that's the hash of the generated kubeadmin [random pass](https://github.com/openshift/installer/blob/8d73989c1efe04448bf62bcbbf0b87b13dbc0a97/pkg/asset/password/password.go#LL44C29-L44C55)?
If so, how is the raw pass exposed to the user? on console?
Or is it something that can be specified by the user on config?

Reply:
Updated to generate the auth assets.